packages/layout/src/components/SideNav/MedlySidenavHeader/MedlySidenavHeader.tsx

```diff
@@ -1,24 +1,18 @@
+import { Logo } from '@medly-components/core';
 import { WithStyle } from '@medly-components/utils';
 import React, { FC, useContext } from 'react';
 import SideNavContext from '../SideNav.context';
-import MedlyLogo from './MedlyLogos/MedlyLogo.svg';
-import MLogo from './MedlyLogos/MLogo.svg';
 import * as Styled from './MedlySidenavHeader.styled';
 import { Props } from './types';
 
-export const MedlySidenavHeader: FC<Props> & WithStyle = React.memo(({ companyLogo: Logo, companyName: Name }) => {
+export const MedlySidenavHeader: FC<Props> & WithStyle = React.memo(({ companyLogo, companyName }) => {
     const { isHovered, isExpanded } = useContext(SideNavContext);
     return (
         <Styled.SidenavHeader as="div" isHovered={isHovered} isExpanded={isExpanded}>
-            <Logo />
-            <Name />
+            <Logo showName={isHovered} name={companyName} logo={companyLogo} />
```

Comment:
`showName` should be true in both the cases `isHovered` or `isExpanded`.

Reply:
Fixed, it is always true now 😄